scripts/package-bin-smoke.mts

```
@@ -668,11 +784,55 @@ function smokeCommandsFor(
         },
       ];
     case "croco":
+      const migrationRunnerInstalled =
+        typeof packageInfo.sourceManifest.dependencies?.["@croco/migration-runner"] === "string";
```

Biome (2.5.7)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)